core/src/main/java/hudson/cli/declarative/CLIRegisterer.java

```diff
@@ -293,7 +293,7 @@ protected int run() throws Exception {
      */
     private ResourceBundleHolder loadMessageBundle(Method m) throws ClassNotFoundException {
         Class c = m.getDeclaringClass();
-        Class<?> msg = c.getClassLoader().loadClass(c.getName().substring(0, c.getName().lastIndexOf('.')) + ".Messages");
+        Class<?> msg = c.getClassLoader().loadClass(c.getName().substring(0, c.getName().lastIndexOf(".")) + ".Messages");
```

Comment:
Hello @basil . Why reverting this?

Reply:
I have retained only the `String#isEmpty` portions of this PR. Other cleanups may be submitted as separate PRs.